src/data/finset/locally_finite.lean

```diff
@@ -373,6 +373,10 @@ lemma Ico_union_Ico_eq_Ico {a b c : α} (hab : a ≤ b) (hbc : b ≤ c) :
   Ico a b ∪ Ico b c = Ico a c :=
 by rw [←coe_inj, coe_union, coe_Ico, coe_Ico, coe_Ico, set.Ico_union_Ico_eq_Ico hab hbc]
 
+@[simp] lemma Ioc_union_Ioc_eq_Ioc {a b c : α} (h₁ : a ≤ b) (h₂ : b ≤ c) :
+  Ioc a b ∪ Ioc b c = Ioc a c :=
```

Comment:
Sorry! Bhavik has been bashing me for four months now for me to write this API. I am now about to do it.